datasets/cartesian/data_generation.py

```diff
@@ -0,0 +1,277 @@
+""" Generating training and test files for iterative decoding on cartesian
+    product problem.
+"""
+
+# Standard library imports
+import numpy as np
+import numpy.random as random
+# Specify local path in which the data will be saved
+data_path = "MyDrive/preprocess_cartesian/token/data/"
+# Flags
+row = False # whether to predict next row or next pair of tokens
+copy = False # whether to copy previous output to next output
+short_input = False # whether to copy only last predicted pair to input or
+                    # entire predicition so far
+# Tokens
+SEP_TOKEN = "[SEP]"
+IN_OUT_TOKEN = "[SEP2]"
+END_TOKEN = "[END]"
+START_TOKEN = "[START]"
+END_ITERATION_TOKEN = "[ENDITER]"
+
+def create_cartesian_dataset(trainsize, testsize, trainmindigits = 1, 
```

Comment:
I think default values should be specified without spaces around the "=" operator according to the style guide. Maybe Josh can confirm (here and the rest of the file)